docs/source/backends/adios2.rst

```diff
@@ -105,6 +105,14 @@ The new layout may be activated **for experimental purposes** in two ways:
 
 The ADIOS2 backend will automatically recognize the layout that has been used by a writer when reading a dataset.
 
+.. tip::
+   This schema does not use ADIOS2 attributes any more, making the ``bpls -a`` and ``bpls -A`` switches useless for openPMD datasets.
+   Their functionality can be emulated via regexes:
+
+   * Print datasets and attributes: Default behavior
+   * Print datasets only: ``bpls -e '.*/__data__$'``
+   * Print attributes only: ``bpls -e '^(.(?!/__data__$))*$'``
```

Comment:
Maybe suggest the user `-l` for long here?
```suggestion
   * Print datasets only: ``bpls -l -e '.*/__data__$'``
   * Print attributes only: ``bpls -l -e '^(.(?!/__data__$))*$'``
```

Reply:
This listing is supposed to show how to select attributes or datasets from a BP file. The `-l` switch doesn't really have anything to do with that.
We can add a little sentence below that the `-l` flag is useful to show the values of scalars?

Reply:
Oh I just add it by muscle memory because I always find it useful, that's why I was wondering if we just doc it that way.